pages/1.13/installing/production/system-requirements/partitions/index.md

```diff
@@ -0,0 +1,50 @@
+
+# Recommended partition layout
+You can use the following guidelines to plan disk partitioning for replicated state stores and persistent configuration override locations under `/var/lib/dcos`.
+
+## Master nodes
+For the master nodes, the recommended practice is to host the `/var/lib/dcos` directory on a separate partition backed by **fast**, **locally-attached** storage (SSD/NVMe). Using this separate partition on the master nodes enables the following replicated state stores and persistent configuration override files to be stored under the `/var/lib/dcos` directory:
+- Mesos Paxos replicated log: /var/lib/dcos/mesos/master/replicated_log
+- Navstar Overlay replicated log: /var/lib/dcos/mesos/master/overlay_replicated_log
+- CockroachDB distributed database /var/lib/dcos/cockroach
+- - Navstar Mnesia distributed database: /var/lib/dcos/navstar/mnesia
+- Navstar Lashup distributed database: /var/lib/dcos/navstar/lashup
+- Secrets vault: /var/lib/dcos/secrets/vault
+- Exhibitor Zookeeper distributed database: /var/lib/dcos/exhibitor/zookeeper
+- History Service cache: /var/lib/dcos/dcos-history
+
+## Agent nodes
+On the agent nodes, you should use separate partitions for the following directories under `/var/lib/mesos`:
+
+- `/var/lib/mesos` - You should always host the `/var/lib/mesos` directory on a separate partition. Keep in mind that the disk space that Apache Mesos advertises in its UI is the sum of the space provided by the file system(s) underpinning the `/var/lib/mesos` directory, including any MOUNT volumes (`/dcos/volume<n>`).
+
+- `/var/lib/mesos/slave/slaves` - This directory hosts the sandbox directories for tasks. You should use a separate partition for this directory, if possible.
+
+- `/var/lib/mesos/slave/volumes` - This directory is used by frameworks that consume ROOT persistent volumes. You should use a separate partition for this directory, if possible.
+
+- `/var/lib/mesos/docker/store` - This directory stores Docker image layers that are used to provision UCR containers. You should use a separate partition for this directory, if possible.
+
+In most cases, agents do not require a separate partition for the persistent configuration override files stored in the `/var/lib/dcos` directory. You should, however, consider using separate partitions for the following directories on agent nodes:
```

Comment:
/var/lib/dcos/mesos-slave-common and /var/lib/dcos/mesos-resources are not directories, they are config files. I don't think they need "separate partitions"

Reply:
Replaced with:
You should, however, be sure to allow enough disk space partitions for the following configuration files on agent nodes: